### ISSUE TITLE
11.2.1 & 11.2.1.1 guarding phrasing

### ISSUE THREAD

**Issue by shannonpileggi** [hadley/r-pkgs#866]
During our discussion of Ch 11, a reader mentioned that the phrasing of guarding was unclear, especially in the context of "11.2.1.1 Whether and how to guard in a test". Its worth considering either adding a defintion for the meaning of guarding or identifying an alternative phrasing, especially for those for whom English is not a first language.

**Comment by jennybc** [comment]
I've looked at the uses of "guard" (currently I find 3) and I feel like they're pretty clear.

I think I'm also OK if reading R Packages leads to someone googling this and/or basically learning the "guard" term in the context of programming.

https://en.wikipedia.org/wiki/Guard_(computer_science)

(I would rephrase if I was just being creative, but I'm not in this case.)
